fadin/solver.py

```diff
@@ -413,7 +315,8 @@ def fit(self, events, end_time):
         return self
 
 
-def plot(solver, plotfig=False, bl_noise=False, title=None, ch_names=None, savefig=None):
+def plot(solver, plotfig=False, bl_noise=False, title=None, ch_names=None,
```

Comment:
This should be moved in a separate file in `utils` in a follow up PR.

Reply:
Something like a `plot.py` file in the utils folder?

Reply:
Done in commit [a7c924e](https://github.com/mind-inria/FaDIn/pull/16/commits/a7c924e3ffd3909338f6b534c4164438636701d8)